### PR TITLE
Create a Script To add Nagios Host Groups

### PR DESCRIPTION
**Pull request by julian55455**
Fixes part of https://github.com/adoptium/infrastructure/issues/2788

This is how it works.

Run the script and enter the related parameters
add2

Halt the nagios server and restart it again using nagios `vagrant halt` for halting and `vagrant up` for starting it again
Login to your nagios webserver in the browser and check for the newly added host group and its members.  You will see a host group called webservers.

add3

cc @steelhead31 @karianna @smlambert @sxa @andrew-m-leonard @Haroon-Khel

### CODE REVIEW

**Review by gdams — COMMENTED** [review]
Not sure how I feel about the filename here... I'd much prefer `nagios_host_group_add.sh`

**Comment by julian55455** [comment]
@gdams sorry I clicked the review button twice. But I changed the file name

File: ansible/playbooks/nagios/roles/Nagios_Server/scripts/nagios_host_group_add.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+echo "enter the hostgroup name"
+read hostgroup_name
+
+echo "enter the hostgroup elias"
```
Comment (karianna) [inline ansible/playbooks/nagios/roles/Nagios_Server/scripts/nagios_host_group_add.sh:6]:
```suggestion
echo "enter the hostgroup alias"
```

File: ansible/playbooks/nagios/roles/Nagios_Server/scripts/nagios_host_group_add.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+read hostgroup_elias
```
Comment (karianna) [inline ansible/playbooks/nagios/roles/Nagios_Server/scripts/nagios_host_group_add.sh:7]:
```suggestion
read hostgroup_alias
```

File: ansible/playbooks/nagios/roles/Nagios_Server/scripts/nagios_host_group_add.sh [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+echo "enter enter the hostgroup members"
+read hostgroup_members
+
+echo "define hostgroup{" >> /usr/local/nagios/etc/objects/hostgroups.cfg
+echo "        hostgroup_name  $hostgroup_name" >> /usr/local/nagios/etc/objects/hostgroups.cfg
+echo "        alias           $hostgroup_elias" >> /usr/local/nagios/etc/objects/hostgroups.cfg
```
Comment (karianna) [inline ansible/playbooks/nagios/roles/Nagios_Server/scripts/nagios_host_group_add.sh:14]:
```suggestion
echo "        alias           $hostgroup_alias" >> /usr/local/nagios/etc/objects/hostgroups.cfg
```

**Review by karianna — CHANGES_REQUESTED** [review]
Made suggested edits

**Review by karianna — APPROVED** [review]

**Review by steelhead31 — APPROVED** [review]
LGTM

**Pull request merged**